backend/wmg/pipeline/summary_cubes/marker_genes.py

```diff
@@ -52,10 +52,12 @@ def create_marker_genes_cube(corpus_path: str):
                 "tissue_ontology_term_ids": [tiss],
                 "cell_type_ontology_term_ids": [ct],
                 "organism_ontology_term_id": organism,
+                "disease_ontology_term_ids": ["PATO:0000461"],
```

Comment:
PAT0:0000461 is the ontology term ID for `normal` cells.

Reply:
worth making that clear via an appropriately named constant